app/templates/organizational-unit/organization/show.emblem

```diff
@@ -1,9 +1,13 @@
-h4 Members
-ul
+= if tabRepositories
+  = each model.repositories as |repo|
+    = if repo.publicAccess
+      = search-result 'unlock-alt' repo.id 'organizationalUnit.repository.show' repo.owner.id repo.repoId
+        p.help-text = repo.description
+        span.label = repo.contentType
```

Comment:
This should be moved into a partial. Maybe in another issue, but it should be done.